Modules/Package.swift

```diff
@@ -85,20 +85,22 @@ let package = Package(
 /// into every target that depends on it. Make sure to avoid including frameworks
 /// with large resources bundled into multiple targets.
 enum XcodeSupport {
-    static let products: [Product] = [
-        .library(name: "XcodeTarget_App", targets: ["XcodeTarget_App"]),
-        .library(name: "XcodeTarget_WordPressTests", targets: ["XcodeTarget_WordPressTests"]),
-        .library(name: "XcodeTarget_WordPressAuthentificator", targets: ["XcodeTarget_WordPressAuthentificator"]),
-        .library(name: "XcodeTarget_WordPressAuthentificatorTests", targets: ["XcodeTarget_WordPressAuthentificatorTests"]),
-        .library(name: "XcodeTarget_ShareExtension", targets: ["XcodeTarget_ShareExtension"]),
-        .library(name: "XcodeTarget_DraftActionExtension", targets: ["XcodeTarget_DraftActionExtension"]),
-        .library(name: "XcodeTarget_NotificationServiceExtension", targets: ["XcodeTarget_NotificationServiceExtension"]),
-        .library(name: "XcodeTarget_Intents", targets: ["XcodeTarget_Intents"]),
-        .library(name: "XcodeTarget_StatsWidget", targets: ["XcodeTarget_StatsWidget"]),
-        .library(name: "XcodeTarget_UITests", targets: ["XcodeTarget_UITests"]),
-    ]
+    static var products: [Product] {
```

Comment:
The previous version seemed a bit more readable.

Reply:
Swift compiler reports a concurrency check error on the previous version. I can't remember the exact error message, but it's related to the variable being a global variable.